src/core/server/events/listeners/webhook.ts

```diff
@@ -12,7 +12,10 @@ export type WebhookCoralEventListenerPayloads = StoryCreatedCoralEventPayload;
 export class WebhookCoralEventListener
   implements CoralEventListener<WebhookCoralEventListenerPayloads> {
   public readonly name = "webhook";
-  public readonly events = [CoralEventType.STORY_CREATED];
+  public readonly events = [
+    CoralEventType.STORY_CREATED,
+    CoralEventType.COMMENT_CREATED,
```

Comment:
I think you may want to also listen for `COMMENT_REPLY_CREATED`? That or I think you can use `COMMENT_ENTERED` and it will trigger for both root comments and replies?